tests/MongoDB.Driver.Core.Tests/Core/Authentication/MongoAWSAuthenticatorTests.cs

```diff
@@ -126,8 +126,6 @@ static MongoAWSAuthenticatorTests()
 
             var actualRequestId0 = sentMessages[0]["requestId"].AsInt32;
             var actualRequestId1 = sentMessages[1]["requestId"].AsInt32;
-            actualRequestId0.Should().Be(expectedRequestId);
```

Comment:
Should this line also be removed?

    var expectedRequestId = RequestMessage.CurrentGlobalRequestId + 1;

Reply:
Good point.
Done.